tests/test_parser_stress.py

```diff
@@ -0,0 +1,350 @@
+from io import BytesIO
+from itertools import chain
+import math
+from numpy import random
+from unittest import TestCase
+
+from requests_toolbelt import MultipartEncoder
+
+from streaming_form_data import StreamingFormDataParser
+from streaming_form_data.targets import ValueTarget
+
+
+def get_random_bytes(size, seed):
+    random.seed(seed)
```

Comment:
Is it possible to fix the seed at the module level? So that this argument is not needed in this function?

Reply:
I have added seed to be an argument to a function specially to have the seed fixed to some value at the beginning of generated buffer.

With the single seed() call the test behavior will change in case we change anything in the middle of tests (lets say if some dataset will change its name or additional dataset will be inserted).